Example/Okta_UITests/Okta_UITests.swift

```diff
@@ -75,15 +68,16 @@ class OktaUITests: XCTestCase {
         app.buttons["Refresh Tokens"].tap()
         app.buttons["Refresh Tokens"].tap()
 
-        sleep(2)
         if let tokenCheck = app.textViews["tokenView"].value as? String {
+            sleep(1)
             XCTAssertNotEqual(oldTokens, tokenCheck)
         }
 
         // Get User info
         app.buttons["Userinfo"].tap()
         if let userInfoValue = app.textViews["tokenView"].value as? String {
-            XCTAssertTrue(userInfoValue.contains("johndoe"))
+            sleep(2)
```

Comment:
Since we are modifying the UI, we need `sleep` methods to allow the UI to update instead of an **Expected Condition**.